README.md

```diff
@@ -14,76 +14,54 @@
   </a>
 </p>
 
-Computational Pathology Toolbox developed by TIA Lab
+Computational Pathology Toolbox developed at the TIA Centre
 
-Getting Started
-===============
+## All Users
 
-First, install
-- OpenSlide [here](https://openslide.org/download/).
-- OpenJPEG [here](https://www.openjpeg.org/), using
-`conda install -c conda-forge openjpeg>=2.3.0`.
+This package is for those interested in digital pathology: including graduate students, medical staff, members of the Tia Centre and of Pathlake, and anyone, anywhere, who may find it useful. We will continue to improve this package, taking account of developments in pathology, microscopy, computing and related disciplines. Send comments and criticisms to **[tia@dcs.warwick.ac.uk](mailto:tialab@dcs.warwick.ac.uk)**.
 
-Git
----
+**`tiatoolbox`** is a multipurpose name that we use for 1) a certain computer program, 2) a Python package of related programs, created by us at the Tia Centre to help people get started in Digital Pathology, 3) this repository, 4) a certain virtual environment.
 
-1.Create the virtual environment
 
-    conda env create -f requirements.conda.yml
-or (on Windows)
+## Developers
 
-    conda env create -f requirements.win64.conda.yml
+Anyone wanting to contribute to this repository, please look [here](https://livewarwickac.sharepoint.com/sites/TIAlab/_layouts/15/Doc.aspx?sourcedoc={1a829baa-2b3e-436a-be6f-b7ea2854cb15}&action=edit&wd=target%28Contributing.one%7C0f5698ce-9ff5-4928-8e09-7004afcbbc87%2FStart%20Here%7Ce55b8846-0037-4917-9911-304566877f9a%2F%29pa), for our naming conventions and other useful information.
```

Comment:
Please add link to wiki instead of OneNote

Reply:
@shaneahmed The only link I could find to Wiki was not publicly accessible. Assuming that Travis has only the permissions of any user anywhere, I would like (to find out how) to write an automated test that goes through all changed files of the repo and detects any web addresses that are not accessible to the general user. I am removing the link to OneNote. I must find out how to add to a "To Do" list. There are two tasks: 1) Relocate the Wiki to a publicly accessible site. 2) Insert  a link in README.md.

Reply:
Have inserted the link suggested by @shaneahmed to the Github Wiki.
I would like the docs-readme branch to be merged now with develop. I will create a new branch in order to remove duplication between README.md and CONTRIBUTING.rst, but there will be a delay before i can get to that.